futurice/futurice.com
# chore: remove templates that are migrated to new content model

- I removed a query field that is no longer used by the template. Leaving it in is causing errors in the netlify build
- removed unused templates
- removed animation libs and deps only used on plan-international-civil-registration-and-vital-statistics-new

## To test
Confirm all these pages still exist
  https://deploy-preview-903--futurice-dev.netlify.app/case-studies/8fit
      https://deploy-preview-903--futurice-dev.netlify.app/case-studies/scandinavian-airlines
      https://deploy-preview-903--futurice-dev.netlify.app/case-studies/halebop
      https://deploy-preview-903--futurice-dev.netlify.app/case-studies/k-ruoka
      https://deploy-preview-903--futurice-dev.netlify.app/case-studies/moneycorp
      https://deploy-preview-903--futurice-dev.netlify.app/case-studies/fortum-apollo
      https://deploy-preview-903--futurice-dev.netlify.app/case-studies/wartsila-weleap
      https://deploy-preview-903--futurice-dev.netlify.app/case-studies/plan-international-civil-registration-and-vital-statistics-new
